Source/WebKit/UIProcess/API/Cocoa/WKPreferences.mm

```diff
@@ -1805,6 +1795,15 @@ - (void)setJavaScriptEnabled:(BOOL)javaScriptEnabled
 
 @implementation WKPreferences (WKPrivateDeprecated)
 
+- (void)_setDNSPrefetchingEnabled:(BOOL)enabled
+{
+}
+
+- (BOOL)_dnsPrefetchingEnabled
+{
+    return NO;
```

Comment:
I think these should be YES now

Reply:
I think they should continue to be false as they were predominantly about the `<a>` feature and control over that through the proprietary header, which we no longer implement after this commit.  And it's not clear to me that we should implement that given the privacy implications.